.github/workflows/ci-aqua-security-trivy-tests.yml

```diff
@@ -1,6 +1,26 @@
 name: ci-aqua-security-trivy-tests
 on:
   push:
+    paths-ignore:
+      - '.vscode/CODEOWNERS'
+      - '.vscode/*.json'
+      - 'compose/**'
+      - 'contrib/**'
+      - 'dgraphtest/**'
+      - 'licenses/**'
+      - 'paper/**'
+      - 'RFC/**'
+      - 'static/**'
+      - 't/**'
+      - 'telemetry/**'
```

Comment:
why should `telemetry` not be in our CI?

Reply:
Cuz there's no test for it at all.

Reply:
There is no unit test in the telemetry package.  It is however imported in two [different](https://github.com/dgraph-io/dgraph/blob/f085c32bae6f7e27475503f64684164f9e3bc085/dgraph/cmd/zero/zero.go#L36) [places](https://github.com/dgraph-io/dgraph/blob/f085c32bae6f7e27475503f64684164f9e3bc085/edgraph/server.go#L56).

Reply:
Yeah. It is imported. But if there's no test for it. I see no reason to run CI. The other packages have their own testing packages. But none of them take telemetry into consideration. It is not a tied relation. But yeah, I think I will remove it. As I have plans to change the telemetry and add tests for that.

This package was added in hurry back them. We were discussing how to get some important data from how ppl use Dgraph. But this was done in a hurry.